integration-test-groups/foundation/scheduler/pom.xml

```diff
@@ -34,6 +34,10 @@
         <dependency>
             <groupId>org.apache.camel.quarkus</groupId>
             <artifactId>camel-quarkus-scheduler</artifactId>
+        </dependency>
+            <dependency>
+            <groupId>org.apache.camel.quarkus</groupId>
+            <artifactId>camel-quarkus-bean</artifactId>
         </dependency>
```

Comment:
I'd vote for doing without this additional dependency. We could e.g. have a an ad hoc processor to increment the counter, something like:

```
.process(e -> myCounter.incrementAndGet());
```